docs/_includes/jlbp-closing-scripts.html

```diff
@@ -0,0 +1,9 @@
+<!-- Closing scripts -->
```

Comment:
What does this script do?

Reply:
@lukesneeringer could you help us understand what the scripts in https://github.com/googleapis/aip/blob/master/_includes/aip-closing-scripts.html do and why we might need them?

Reply:
- The inline one makes the header bar work on mobile.
- The other two are what Glue tells you to use. https://glue-docs.appspot.com/get-started/cdn/

Reply:
I will add comments to the file explaining this.